docs/accordproject-tour.md

```diff
@@ -3,29 +3,29 @@ id: accordproject-tour
 title: Online Tour
 ---
 
-To get an better acquainted with Accord Project templates, the easiest way is through the online [Template Studio](https://studio.accordproject.org) editor.
+To get an better acquainted with Accord Project templates, the easiest way is through the online [Template Playground](https://studio.accordproject.org) editor.
 
 :::tip
-You can open template studio from anywhere in this documentation by clicking the [Try Online!](https://studio.accordproject.org) button located in the top-right of the page.
+You can open Template Playground from anywhere in this documentation by clicking the [Try Online!](https://studio.accordproject.org) button located in the top-right of the page.
 :::
 
-The following video offers a tour of Template Studio and an introduction to the key concepts behind the Accord Project technology.
+The following video offers a tour of Template Playground and an introduction to the key concepts behind the Accord Project technology.
```

Comment:
We should probably just remove this page entirely since it is focused on an outdated video.